octoprint_mrbeam/__init__.py

```diff
@@ -738,6 +739,15 @@ def calibration_tool_mode(self):
         self._fixEmptyUserManager()
         return ret
 
+    @property
+    def user_checks_for_updates(self):
+        return self._user_checks_for_updates
+
+    @user_checks_for_updates.setter
+    def user_checks_for_updates(self, value):
```

Comment:
Since it is a boolean value, I would suggest to create tow setters, one for set and one for clear

Reply:
ok i can change that

Reply:
will change it to set_explicit_update_check and clear_explicit_update_check